src/content/reference/react-dom/components/form.md

```diff
@@ -0,0 +1,445 @@
+---
+title: "<form>"
+canary: true
+---
+
+<Canary>
+
+React's `form` Component is currently only available in React's canary and experimental channels but you can use the [standard HTML `form` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) in your React's stable channel. Learn more about [React's release channels here](/community/versioning-policy#all-release-channels).
+
+</Canary>
+
+
+<Intro>
+
+The [built-in browser `<form>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) lets you create interactive controls for submitting information.
+
+```js
+<form action={search}>
+    <input name="query" />
+    <button type="submit">Search</button>
+</form>
+```
+
+</Intro>
+
+<InlineToc />
+
+---
+
+## Reference {/*reference*/}
+
+### `<form>` {/*option*/}
+
+The [built-in browser `<form>` component](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) lets you create interactive controls for submitting information.
+
+```js
+<form action={search}>
+    <input name="query" />
+    <button type="submit">Search</button>
+</form>
+```
+
+[See more examples below.](#usage)
+
+#### 🚧 Props {/*props*/}
+
+`<form>` supports all [common element props.](/reference/react-dom/components/common#props)
+
+Additionally, `<form>` supports these props:
+
+* [`action`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#disabled): A boolean. Specifies what to do when a form is submitted. The form data can be handled with client or server side code.
+* 🚧
+
+
+🚧 Forms can contain one or more of the following tags:
+* [`<input>`]
```

Comment:
Oof, it's hard to know whether to list everything that the browser supports here, or only the things that have special React behavior. How are you thinking about that?

Reply:
I was starting by blindly copying what other DOM component pages are doing like [input](https://react-kbffi0laq-fbopensource.vercel.app/reference/react-dom/components/input). I know this is different because its supported tags inside `form` instead of attributes of `form` but that's why this list is here, so I don't forget to address this. Its going to be a little tricky, we'll at least need to document submit buttons pretty well because of `formAction`. I haven't given it much thought yet overall though, this is mostly a placeholder for me to remember to address this.

Reply:
My view is that we should only document the differences and just link to MDN for the standard stuff. Otherwise, the standard stuff drowns out the signal of what's different, while also not being a very good reference for the standard stuff because we're never going to go into as much detail as MDN does.

Reply:
I removed this list